menpo/shape/groupops.py

```diff
@@ -1,19 +1,20 @@
 from __future__ import division
-from .pointcloud import PointCloud
 
 
 def mean_pointcloud(pointclouds):
     r"""
-    Compute the mean of a `list` of :map:`PointCloud` objects.
+    Compute the mean of a `list` of :map:`PointCloud` or subclass objects.
```

Comment:
may want to mention that the list is assumed to be homogenous (since we just blindly take the first element of the list and I guess assume all items in the list are of the same PointCloud subclass)